src/main/java/com/pedjak/gradle/plugins/dockerizedtest/TestExecuter.java

```diff
@@ -1,86 +0,0 @@
-package com.pedjak.gradle.plugins.dockerizedtest;
```

Comment:
Unfortunatelly, we need to have our own `TestExecuter` because we need to instantiate our own `ForkingTestClassProcessor`. Gradle internal architecture is not modular enough, so we cannot specify our own `ForkingTestClassProcessor` somewhere on `Test` task, but we can our own `TestExecuter`. The reason we need our own `ForkingTestClassProcessor` is to achieve a bit more robustness before failing the whole test task. It can happen that there are some failure at the time when we spin up a test container, but these errors are intermediate. Thus we try to start the test worker in a container a few times before giving up.